admin/services/pin-generation.service.js

```diff
@@ -80,7 +61,7 @@ pinGenerationService.generateSchoolPassword = school => {
     wordsArray[pinGenerationService.generateCryptoRandomNumber(0, wordsArray.length - 1)]
   const numberCombination = randomGenerator.getRandom(2, chars)
   const newPin = `${firstRandomWord}${numberCombination}${secondRandomWord}`
-  const newExpiry = pinExpiryTime(school.timezone)
+  const newExpiry = pinValidityGeneratorService.generatePinTimestamp(config.OverridePinExpiry, endOfDay(), fourPmToday(), school.timezone)
```

Comment:
I have to admit that I am confused by the meaning of `pinValidityGeneratorService`.  Can validity be generated?  Also, in comp-sci, `generator` often has a specific meaning, which is not meant here.

If we remove all the extra words here, I see `pinService`.  I think this is simpler and more direct.

Reply:
moved to `pinService`